PocketKit/Sources/PocketKit/Home/CompactHomeCoordinator.swift

```diff
@@ -252,6 +257,13 @@ class CompactHomeCoordinator: NSObject {
         guard !isResetting, let alert = alert else { return }
         viewController.present(UIAlertController(alert), animated: !isResetting)
     }
+    
+    func present(_ viewModel: AddTagsViewModel?) {
+        guard let viewModel = viewModel else { return }
```

Comment:
When presenting modal view controllers, we need to account for the case where a user is changing from compact to regular. Since modal view controllers persist during the transition, we want to "short circuit" this logic so we don't present a second time. 

TLDR; we need to `guard !isResetting else { return }` here 😆

Reply:
oh thanks! I tried to replicate this issue but wasn't sure how to, but I updated the change. We can discussed next week on how to reproduce the issue.